packages/@okta/vuepress-site/docs/guides/event-hook-ngrok/main/index.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Event Hooks with ngrok
+excerpt: How to demonstrate Event Hooks using ngrok to expose a local app to the internet.
+layout: Guides
+---
+
+This guide demonstrates how to develop and test Okta Event Hooks with the ngrok utility.
+
+Event Hooks are outbound calls from Okta that can be used to notify your own software systems of events occurring in your Okta org. See [Event Hooks](/docs/concepts/event-hooks/) for an overview.
+
+The ngrok utility exposes local web servers to the internet, and can enable the testing of Okta Event Hooks using a local application, rather than an internet-based production or test external service.
+
+To use this guide, you need the following:
+
+- An Okta developer org. [Create an org for free](https://developer.okta.com/signup/).
+- A local application. Or use the one provided in the following guide.
+- The ngrok utility. Installation details are provided in the following guide.
+
+With those prerequisites satisfied, use the following steps to get up-and-running:
+
+1. [Install ngrok](/#install-ngrok)
+1. [Create a local sample app](/#create-a-local-application)
+1. [Set up an Event Hook](/#create-an-okta-event-hook)
+1. [Preview and test](/#preview-test-and-review-the-event-hook)
```

Comment:
These 4 anchor links didn't seem to work for me when I ran this on my local machine. For example clicking on 1st link sent me to
http://localhost:8080/#install-ngrok

instead of
http://localhost:8080/docs/guides/event-hook-ngrok/nodejs/main/#install-ngrok

Reply:
Thanks. I've removed these links as we're using a different "architecture" for our guides now.